phpstan.neon

```diff
@@ -0,0 +1,16 @@
+includes:
```

Comment:
Should we name this `phpstan.neon.dist` as per https://phpstan.org/config-reference#config-file?

Reply:
Thanks! Changed, rebased and repushed.